### PR TITLE
Update meta.yaml and run `conda smithy rerender`

### PR DESCRIPTION
**Pull request by wd15**
See https://github.com/conda-forge/conda-forge.github.io/issues/303

### CODE REVIEW

**Comment by isuruf** [comment]
It seems you need scipy in build requirements.

**Comment by isuruf** [comment]
You need matplotlib, pyparsing and pytables in build requirements

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,13 +13,16 @@ source:
 
 build:
   number: 0
-  skip: true
```
Comment (isuruf) [inline recipe/meta.yaml:16]: Can you make this `skip: true   # [py>35]` to skip python3.6 as not all your dependencies are on python3.6 yet?

Comment (isuruf) [reply recipe/meta.yaml:16]: You'll need to rerender after that

**Comment by wd15** [comment]
@isuruf it looks like things have passed, shall I go ahead and merge?

**Review by isuruf — APPROVED** [review]
LGTM

**Pull request merged**